(ForumsTeam) Cannot create classic Vms from classic portal.

JainNirushi opened the pull request:
From this document https://docs.microsoft.com/en-us/azure/virtual-machines/windows/classic/troubleshoot-recovery-disks-portal , the suggestions outlined to create classic VMs from classic portal is no more available. The document has to be updated to describe the procedure using Azure portal and also add similar suggestions/process for ARM VMs.
Specifically," Create a VM from the original hard disk" in the document.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Blackmist commented:
Looks good to merge.

JainNirushi committed:
(ForumsTeam) Cannot create classic Vms from classic portal.

From this document https://docs.microsoft.com/en-us/azure/virtual-machines/windows/classic/troubleshoot-recovery-disks-portal , the suggestions outlined to create classic VMs from classic portal is no more available. The document has to be updated to describe the procedure using Azure portal and also add similar suggestions/process for ARM VMs.
Specifically," Create a VM from the original hard disk" in the document.
